logo/logo.R

```diff
@@ -0,0 +1,43 @@
+library(ggplot2)
```

Comment:
Please remove the script generating the logo, the code is not a part of the package ;)